src/test/java/io/keycloak/builder/Utils.java

```diff
@@ -2,17 +2,78 @@
 
 import static io.quarkus.bootstrap.util.PropertyUtils.isWindows;
 
-import java.io.File;
-import java.io.IOException;
+import java.io.*;
+import java.nio.file.Path;
+
 import org.apache.commons.io.FileUtils;
+import org.testcontainers.containers.PostgreSQLContainer;
 
 public class Utils {
 
+    String SCRIPT_CMD = isWindows() ? "kc.bat" : "kc.sh";
+    String SCRIPT_CMD_INVOKABLE = isWindows() ? SCRIPT_CMD : "./" + SCRIPT_CMD;
+
+    static String postgresPassword="password";
+    static String postgresUsername="keycloak";
+    static String postgresDbName="keycloak";
+    static PostgreSQLContainer POSTGRES_CONTAINER;
+    static String postgresJDBCUrl;
+
+    public static void startPostgreContainer() throws IOException, InterruptedException {
+        final String POSTGRES_DOCKER_IMAGE_NAME = "postgres:latest";
+        POSTGRES_CONTAINER = new PostgreSQLContainer(POSTGRES_DOCKER_IMAGE_NAME);
+        POSTGRES_CONTAINER
+                .withDatabaseName(postgresDbName)
+                .withUsername(postgresUsername)
+                .withPassword(postgresPassword)
+                .start();
+        postgresJDBCUrl = POSTGRES_CONTAINER.getJdbcUrl().replace("jdbc:postgresql://","");
+    }
+
+    public static void readProcessLogs(Process process){
+        BufferedReader br = new BufferedReader(new InputStreamReader(process.getInputStream()));
```

Comment:
This reader should be closed. It could be moved to
````
try(BufferedReader br = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
````

Reply:
ah, you are not waiting until the process terminates here

Reply:
I'd suggest reading the whole output of the process